tendrl/gluster_integration/objects/definition/gluster.yaml

```diff
@@ -60,6 +60,17 @@ namespace.gluster:
       type: Create
       uuid: b8dd2b4a-96de-4ded-b17d-374d26ec7593
       version: 1
+    SyncAggregatedClusterDetails:
+      tags:
+        - "provisioner/gluster"
```

Comment:
I am ok with that as long as its run only once and that is taken care by making sure its started from the node with tag `provisioner/gluster`. @r0h4n comments??